test/EntityFramework/FunctionalTests/ProductivityApi/SimpleScenarios.cs

```diff
@@ -15,7 +15,6 @@ namespace ProductivityApiTests
     ///     They are here as a good starting point for understanding the functionality and to give some level
     ///     of confidence that the simple scenarios keep on working as the code evolves.
     /// </summary>
-    [PartialTrustFixture]
```

Comment:
@ajcvickers Is it OK to remove the partial trust tests or should I port them too?

Reply:
Remove them!